perun-dispatcher/src/main/java/cz/metacentrum/perun/dispatcher/scheduling/impl/SchedulingPoolImpl.java

```diff
@@ -146,7 +147,24 @@ public void setPerun(Perun perun) {
 		this.perun = perun;
 	}
 
-
+	// --- session init ----------------------------------
+	
+	public void afterPropertiesSet() {
+		// init session
+		try {
+			if (sess == null) {
+				sess = perun.getPerunSession(new PerunPrincipal(
+								dispatcherProperties.getProperty("perun.principal.name"),
+								dispatcherProperties.getProperty("perun.principal.extSourceName"),
+								dispatcherProperties.getProperty("perun.principal.extSourceType")),
+						new PerunClient());
+			}
+		} catch (InternalErrorException e1) {
+			log.error("Error establishing perun session to add task schedule: ", e1);
+			return;
```

Comment:
This `return` is not needed, since it is the last call of the method.

Reply:
Removed.